### ISSUE TITLE
Don't highlight `value` as keyword if in a non-setter method.

### ISSUE THREAD

**Issue by nielsdg** [arrufat/vala.vim#2]
The `value` keyword is used in auto-generated setters and is appropriately highlighted. However, it is also highlighted in non-setter methods, leading to a bit of confusion.

**Comment by arrufat** [comment]
Yes, I have to figure out how to do this, as well as highlighting methods that start with `@`...
